app/src/main/java/com/github/orkest/domain/FireStoreDatabaseAPI.kt

```diff
@@ -311,6 +299,47 @@ class FireStoreDatabaseAPI {
         return future
     }
 
+
+    //TODO: Add the condition on posts belonging to the user's friends
+    /**
```

Comment:
Very good remark. I'm not sure it would be implemented in the function of fetching the posts form the database though but rather when calling the function, we would need to call it only for the user's followings.

Reply:
I am not sure I understood what you mean, the fetching of recents posts is independent of the username right now

Reply:
Yes I know, I am making a remark to the TODO you added at the top of the getRecentPostsFromDatabase.

Reply:
Yes I know, but I didn’t understand what you meant, right now calling the function is not user specific

Reply:
Because you wrote the TODO at the top of the getRecentPostsFromDatabase, I understood that you wanted to make the function user specific (in a future sprint maybe) to implement the modification. That's why I made that remark :)